src/NServiceBus.Transport.Msmq/MsmqSettings.cs

```diff
@@ -50,6 +51,11 @@ public MsmqSettings(ReadOnlySettings settings)
                 TimeToReachQueue = timeToReachQueue;
             }
 
+            if (settings.TryGet<TimeSpan>("NServiceBus.Transport.Msmq.MessageEnumeratorTimeout", out var messageEnumeratorTimeout))
```

Comment:
Why is this prefixed with `NServiceBus.Transport.Msmq.` compared to the other settings?

Reply:
To make sure it’s unique, we should change the others in the long term